src/main/app/src/components/toteutus/ToteutusHakukohteet.tsx

```diff
@@ -0,0 +1,298 @@
+                            wrap="nowrap"
+                            alignItems="flex-start">
+                            <Grid item>
+                              <Typography className={classes.gridHeading} noWrap>
+                                {t('koulutus.pohjakoulutus') + ':'}
+                              </Typography>
+                            </Grid>
+                            {!_.isEmpty(haku.pohjakoulutusvaatimusTarkenne) && (
+                              <Grid item>
+                                <LabelTooltip
+                                  title={
+                                    <LocalizedHTML
+                                      noMargin
+                                      data={haku.pohjakoulutusvaatimusTarkenne}
+                                    />
+                                  }
+                                />
+                              </Grid>
+                            )}
+                          </Grid>
+                          <Grid item>
+                            <Typography variant="body1" noWrap>
+                              {haku.pohjakoulutusvaatimus.map(l.localize)}
+                            </Typography>
+                          </Grid>
+                        </Grid>
+                        {haku.aloituspaikat && (
+                          <Grid item xs={6}>
+                            <Grid item>
+                              <Typography className={classes.gridHeading} noWrap>
+                                {t('toteutus.opiskelupaikkoja') + ':'}
+                              </Typography>
+                            </Grid>
+                            <Grid item>
+                              <Typography variant="body1" noWrap>
+                                {haku.aloituspaikat}
+                              </Typography>
+                            </Grid>
+                          </Grid>
+                        )}
+                      </Grid>
+                    </Grid>
+                    <Grid item>
+                      <ButtonGroup
+                        className={classes.lomakeButtonGroup}
+                        orientation="horizontal"
+                        color="primary">
+                        {haku.hakulomaketyyppi !== HAKULOMAKE_TYYPPI.EI_SAHKOISTA && (
+                          <Button
+                            variant="contained"
+                            size="large"
+                            color="primary"
+                            target="_blank"
+                            href={l.localize(haku.hakulomakeLinkki)}
+                            disabled={!haku.isHakuAuki}>
+                            <Typography style={{ color: colors.white }} variant="body1">
+                              {t('toteutus.tayta-lomake')}
+                            </Typography>
+                          </Button>
+                        )}
+
+                        <Button
```

Comment:
Tässä oli alunperin LocalizedLink lapsenaan Button, joka sitten heitti erroria tietty.

En lähteny purkaan tuota rakennetta vaan käänsin oikeinpäin komponentit, mutta ei tästä kaunis tullu.